concepts/string.md

```diff
@@ -25,12 +25,14 @@ So a string can even be used for a paragraph or book.
 
 ## Encoding
 
-In software, every character has a unique number.
+Computers deal with numbers, not with characters.  In order to represent a character, it has to be encoded into a series of bits.  Since computers deal with bytes (8 bits), a character is typically encoded into one or several bytes of data.
```

Comment:
I'd phrase it as:

> At the low level, computers use numbers, not characters.  So each character is actually represented by a unique number.

This part is optional, in my view:
> For a character to be represented with a number, it is encoded as one or more bytes.  Each byte is a sequence of bits.  Each bit is 0 or 1.

Because the main point for this audience is that each character gets a number, like 65.

The fact that 65 in decimal can also be represented as 1000001 in binary or 41 in hex is irrelevant.

Because that's explaining how integers are encoded, which is out of scope.

In any case, we could add a column to the table to show the binary.